### PR TITLE
Include Security Article in Documentation 

### PR DESCRIPTION
**Pull request by CompeyDev**
As discussed, instead of including a permissions system in lune (https://github.com/lune-org/lune/issues/25), the better alternative would be to educate users instead. 

This PR includes a pure luau sandbox script and a security article explaining what to do with untrusted scripts.

### CODE REVIEW

**Review by filiptibell — CHANGES_REQUESTED** [review]
The security article looks great 👍 but there are several unrelated changes blocking this from getting merged:

- Reformatting of settings / manifest files and driveby changes to versions
- Changes in autogenerated API docs, right now the generator does not work with the new regex apis, I'll take a look and fix it, but it also means the new generated docs are incorrect and must not be published (which would happen automatically on merge)
- Changing the package manager to pnpm for some reason?

If the above changes are reverted and the PR is just the security article I can merge it in

**Comment by CompeyDev** [comment]
> Reformatting of settings / manifest files and driveby changes to versions

Wasn't aware that they were reformatted, I just bumped the lune version since the sandbox script requires *all* of the standard libraries for the latest version of lune.

> Changing the package manager to pnpm 

I have npm aliased to pnpm since npm gets too slow on large repositories, must've been that.

**Comment by CompeyDev** [comment]
@filiptibell Could you remove the `.gitattributes` file once this is merged?

**Review by filiptibell — APPROVED** [review]

**Pull request merged**
